docs/developers-guide/e2e-tests.md

```diff
@@ -187,6 +187,33 @@ Embedding SDK is a library, and not an application. We use Storybook to host pub
 
 In order to run stories used for tests locally, please check [storybook setup docs](https://github.com/metabase/metabase/blob/master/enterprise/frontend/src/embedding-sdk/README.md#storybook)
 
+### Sample Apps compatibility with Embedding SDK tests
+
+In order to check compatibility between Sample Apps and Embedding SDK, we have a special test suite for each sample app that pulls this Sample App, starts it and runs its Cypress tests against the local `metabase.jar` and local `@metabase/embedding-sdk-react` package.
+
+To run these tests locally, run:
+```
+CYPRESS_ALL_FEATURES_TOKEN=<token> TEST_SUITE=<sample_app_repo_name>-e2e OPEN_UI=false EMBEDDING_SDK_VERSION=local START_METABASE=false GENERATE_SNAPSHOTS=false START_CONTAINERS=false yarn test-cypress
```

Comment:
I think `CYPRESS_ALL_FEATURES_TOKEN` is derived from `ENTERPRISE_TOKEN, in `run_cypress_local.ts`, so we can set `ENTERPRISE_TOKEN` instead:

```
// if you want to change these, set them as environment variables in your shell
const userOptions = {
  ENTERPRISE_TOKEN: "",
}

const derivedOptions = {
  CYPRESS_ALL_FEATURES_TOKEN: userOptions.ENTERPRISE_TOKEN,
}
```